lib/rubocop/cop/style/space_after_colon.rb

```diff
@@ -12,21 +12,35 @@ class SpaceAfterColon < Cop
 
         def on_pair(node)
           oper = node.loc.operator
-          return unless oper.is?(':') && followed_by_space?(oper)
 
-          add_offense(oper, oper)
+          return unless oper.is?(':')
+
+          add_offence_if_colon_not_followed_by_space(oper)
         end
 
         def on_if(node)
           return unless ternary_op?(node)
 
-          colon = node.loc.colon
-          return unless followed_by_space?(colon)
+          add_offence_if_colon_not_followed_by_space(node.loc.colon)
+        end
+
+        def on_kwoptarg(node)
+          name = node.loc.name
+          # The name source range is inclusive of the variable name and colon.
+          colon = Parser::Source::Range.new(
```

Comment:
Whoops, I meant I prefer the *former* I.e. with the single `^` pointing to the colon!

Indeed, both yield the same result from `#not_followed_by_space?` (that's what I was trying to get at when I said " but I think 1. would work fine with the range copying/extending as you suggest")

So, I propose to *not* make the change to something like `name.resize(name.size + 1)` - are you ok with this?